storage-node/src/commands/server.ts

```diff
@@ -350,7 +350,11 @@ Supported values: warn, error, debug, info. Default:debug`,
         },
         x_host_id: X_HOST_ID,
       })
-      app.listen(port, () => logger.info(`Listening on http://localhost:${port}`))
+      const server = app.listen(port, () => logger.info(`Listening on http://localhost:${port}`))
+
+      // INFO: https://nodejs.org/dist/latest-v18.x/docs/api/http.html#serverrequesttimeout
+      // Set the server request timeout to 0 to disable it. This was default behaviour pre Node.js 18.x
+      server.requestTimeout = 0
```

Comment:
didn't expected this change in this PR but I guess it's expected right @mnaamani ?

Reply:
The reason we see it here, is because I've included updates from master branch. Because I know QA testing will happen from the luxor branch so best for it to also have what will ultimately be the proper version used for all the apps.

Reply:
That's because the PR includes updates from master - https://github.com/Joystream/joystream/commit/8ab3a4342e54e65d12006495436f413974493a76